app/assets/javascripts/models/ui/reverb_retirement.coffee

```diff
@@ -5,14 +5,16 @@ ns = @edsc.models.ui
 data = @edsc.models.data
 
 ns.ReverbRetirement = do (ko,
-                          ajax = @edsc.util.xhr.ajax) ->
+                          ajax = @edsc.util.xhr.ajax,
+                          config=@edsc.models.data.config) ->
   class ReverbRetirement
     constructor: ->
 
     referrerIsReverb: () =>
       console.log "Checking referrer: " + document.referrer
       referrer = if document.referrer then document.referrer.match(/:\/\/(.[^/]+)/)[1] else false
-      reverb = ["echo-reverb-rails.dev", "testbed.echo.nasa.gov", "api-test.echo.nasa.gov", "testbed.echo.nasa.gov", "reverb.echo.nasa.gov"]
+      reverb = [edsc.config.reverb_url]
```

Comment:
```reverb``` will no longer be an array. For each env, only one URL is valid. For example, if current env is SIT, reverb url will start with https://testbed.echo.nasa.gov